services/frontend-service/src/ui/utils/store.tsx

```diff
@@ -306,7 +307,7 @@ export const useEnvironmentLock = (lockId: string) =>
         )?.name,
     } as DisplayLock);
 
-export const searchCustomFilter = (queryContent: string | null, val: string | undefined) => {
+export const searchCustomFilter = (queryContent: string | null, val: string | undefined): string | undefined | null => {
```

Comment:
Can we handle the `undefined | null` cases here and return perhaps `''`?

Reply:
done